README.md

```diff
@@ -1,2 +0,0 @@
-# Frontend
-This repo will be linked to Netlify for ease of deployment and code review
```

Comment:
Why did you delete the README.md?

Reply:
oh because we will be replacing it with the new readme.md that's currently in a pull request

Reply:
It will prevent a merge conflict later on

Reply:
If this is merged first, then when you pull the master branch into the new README branch it suspect there will be a conflict. If it is merged second, then I suspect it will remove the README

Reply:
Ah true that's a good point, I shall revert the readme deletion.